src/docs/design/design-audit.mdx

```diff
@@ -0,0 +1,210 @@
+
+<ImageGrid
+    images={[Educational1, Educational2]}
+    captionsUnderImages={[
+        `AVOID: Avoid giving the team an instruction without explaining the rationale behind it.`,
+        `DO: Teach the team how this design decision is made.`,
+    ]}
+/>
+
+For larger products, instead of trying to cover everything in superficial detail, auditors may select 2-3 key interactions and examine it in depth. The auditors’ mission should not be to point out every single UI mistake in this case, as there are likely going to be follow-up UI audits in the future, but to teach the team how to avoid similar mistakes in the future.
+
+## Prototype Key Features
+
+If time permits, auditors may select a few key features and prototype their design recommendation. This has a few advantages.
+
+1. Auditors understand better why the product team designed a feature in a certain way.
+2. Auditors get a good design exercise.
+3. A new design placed next to its original design serves as a good side-by-side comparison.
+
+## Summarize Takeaways
+
+Auditors are encouraged to provide an overall summary of their audit findings. This provides teams quick takeaways that helps them estimate their effort in improving the UI design.
+
+## Presentation
+
+Auditors may tune their presentation depending on who they will be presenting to. For example, if the audiences possess a more technical background, consider explaining the styling in CSS notations; if they are more on the product design side, consider showing your raw design process and inviting them for a design critique.
+
+Some team members might be daunted by the auditor’s authority in design; however, they have been with the project way longer than the auditor and likely know some legacy or technical reasons for why things are done in a certain way. Auditors should encourage teams to challenge the design audit result to foster a better discussion.
+
+Auditors need to remember to pause from time to time in their presentations for the audiences to catch up — likely not many people in the audience are used to thinking the way a UI designer thinks. Auditors should also ask one of the audience to take notes.
+Recommend Next Steps
+Before the end of their presentations, auditors should recommend the next steps for teams to act upon. If the auditor has a technical background, they may list a few items as easy-to-fix quick-win items for the developer. If there are major feature suggestions, the auditor may picture how they will fit into the product’s big picture, and what the R&D timeline would look like.
+
+## Improve Brightlayer UI
+
+During the audit, if auditors noticed anything worth componentizing, or that the existing Brightlayer UI standard is too strict, please suggest it to us. Many design guidelines in Brightlayer UI today came from real project design audits we did.
```

Comment:
```suggestion
During the audit, if auditors noticed anything worth componentizing or that the existing Brightlayer UI standard is too strict, please suggest it to us. Many design guidelines in Brightlayer UI today came from real project design audits we did.
```

Reply:
Some of my comments around removing the pauses comes down to _how_ we wants the docs to read, I guess more of a voice suggestion.  If this is a theme that we have going on other pages, where we add pauses between parts of our sentence, feel free to ignore those suggestions.